app/views/index.scala.html

```diff
@@ -7,9 +7,28 @@
 
 @main(title) {
     @navbar(user)
+    <script src='@routes.Assets.at("assets/detectMobileBrowser.js")'></script>
     <script src='@routes.Assets.at("assets/homepage.js")'></script>
+    <!--<script>
```

Comment:
@sbower213 If this commented out code is not required, then could you remove it? If commented out for a reason, then could you add that before it?